src/components/buttons/MiniButton.js

```diff
@@ -38,10 +38,17 @@ const Content = styled(RowWithMargins).attrs({
   align: 'center',
   margin: 5,
 })`
-  ${({ hasLeadingIcon, small }) =>
-    padding(0, small ? 8 : 10, 0, hasLeadingIcon ? 6 : small ? 8 : 10)};
+  ${({ hasLeadingIcon, small, disablePadding }) =>
```

Comment:
Hey, could you please rebase this PR? We no longer support css strings with styled components as well as we should only use `@rainbow-me/styled-components` package.

See the content of this file in develop:
https://github.com/rainbow-me/rainbow/blob/develop/src/components/buttons/MiniButton.js

Reply:
I guess you should rebase the whole register-ens branch.
Sorry for adding some extra work, but this is required to make sure our app is fast on android.

Reply:
Oh yep, once I rebase, this should be gone!